app/controllers/student/response_controller.rb

```diff
@@ -24,23 +24,23 @@ def show
 
```

Comment:
Getting all of the setup for this done early has made it nice for you to go back and fill in =)